### DISCUSSION TITLE
Guide for using this tool with DTLS

### DISCUSSION THREAD

**Discussion by satyam-m** [radcli/radcli#86] [General]
Can someone brief:
1. how do we use this client if we want to send radius message securely using DTLS.
2. What is expected to be implemented on server side for the same.

Finding it hard to find it in doc which clearly answers both the queries.

Would be helpful if you can please let me know on above.

**Comment by nmav** [comment]
There are no code changes, just configuration changes. Check the example configuration file at:
https://radcli.github.io/radcli/manual/radiusclient_8conf-example.html

and in particular the `serv-type` option.
